deps.edn

```diff
@@ -6,6 +6,7 @@
         io.replikativ/konserve                      {:mvn/version "0.7.285"}
         persistent-sorted-set/persistent-sorted-set {:mvn/version "0.2.1"}
         environ/environ                             {:mvn/version "1.2.0"}
+        cheshire/cheshire                           {:mvn/version "5.11.0"}
```

Comment:
Something failed without it up there. I can find out what it was

Reply:
The cljdocs preview fails without the package here because the datahike.cli is mixed in with the rest of the datahike namespaces I guess. We could and maybe even should change that when changing to a polylith structure.

Reply:
the cljdoc preview fails like it always did. I never got it to work. Let's see if cljdoc works with a new release. It seems to me as if it was flaky. sometimes it works.